src/mainviews/job-detail.tsx

```diff
@@ -1,68 +1,53 @@
-import React, { useEffect, useState } from 'react';
+import React from 'react';
```

Comment:
If this file is intended for use only in the detail view, perhaps it should be at `mainviews/detail-view/job.tsx`

Reply:
See above.